pkgs/development/ocaml-modules/mdx/default.nix

```diff
@@ -17,7 +17,10 @@ buildDunePackage rec {
   };
 
   nativeBuildInputs = [ cppo ];
-  propagatedBuildInputs = [ astring fmt logs csexp ocaml-version odoc-parser re findlib ];
+  propagatedBuildInputs = [
+    (logs.override { jsooSupport = false; lwtSupport = false; })
```

Comment:
This should be done in a let in and be named logs' to avoid using two variants of logs by accident and making this list rather big

Reply:
Changed. I also found that I got the enable flags wrong, fixed too.